services/admin.js

```diff
@@ -0,0 +1,44 @@
+const User = require('~/models/user')
+const Role = require('~/models/role')
+const {
+  roles: { ADMIN }
+} = require('~/consts/auth')
+const { USER_NOT_REGISTERED } = require('~/consts/errors')
+const { createError } = require('~/utils/errorsHelper')
+
+const adminService = {
+  getAdmins: async () => {
+    const role = await Role.findOne({ role: ADMIN }).lean().exec()
+    const admins = await User.find({
+      role: role._id
+    }).lean().exec()
+
+    return admins.map((admin) => {
+      return {
+        id: admin._id,
+        firstName: admin.firstName,
+        lastName: admin.lastName,
+        email: admin.email
+      }
+    })
+  },
+
+  getAdmin: async (userId) => {
+    const role = await Role.findOne({ role: ADMIN }).lean().exec()
+    const admin = await User.findOne({
+      _id: userId,
+      role: role._id
+    }).lean().exec()
+
+    if (!admin) throw createError(404, USER_NOT_REGISTERED)
+
+    return {
+      id: admin._id,
```

Comment:
also here

Reply:
I cannot use destructuring on the admin variable, as it may be null or undefined.

Reply:
you can use on 32 line

Reply:
You want me to destructure the object after the check?

Reply:
fixed